### ISSUE TITLE
AMDeadTimeButtons don't appear for BioXAS applications running on Mac.

### ISSUE THREAD

**Issue by helfrij** [acquaman/acquaman#2231]
I just happened to notice that the dead time buttons in the XRF detector view don't appear in the view for BioXAS Side and Main applications when the application is running on my Mac. The buttons still take up the same amount of space and can still be interacted with (eg. tooltips appear, can be clicked), they just appear to be styled as the background. I think they used to appear (?). 

There were some recent changes to this class (#2191) that could have accidentally caused this behavior.

**Comment by dretrex** [comment]
Hey @helfrij, I just wanted to confirm that they do not appear on my mac either.

**Comment by helfrij** [comment]
I've added some QDebugs to the master version of AMDeadTimeButton, and we seem to be falling into the correct if-statements to appropriately style the buttons. I've also checked out an old version of the AMDeadTimeButton classes (from branch #2202), before the ICR modifications were made in #2191, and the buttons still don't appear. I guess it's possible that the buttons stopped appearing before the ICR modifications were made? Hmmm.
